### PR TITLE
#285: Fix hardcoded 30s timeout.

### PR DESCRIPTION
**Pull request by intelradoux**
Hello,

A small fix to link the initDB timeout to the existing adbTimeout.

### CODE REVIEW

File: spoon-runner/src/main/java/com/squareup/spoon/SpoonUtils.java [hunk 1]
```diff
@@ -129,8 +129,8 @@ static void createAnimatedGif(List<File> testScreenshots, File animatedGif) thro
     encoder.finish();
   }
 
-  private static void waitForAdb(AndroidDebugBridge adb) {
-    long timeOutMs = TimeUnit.SECONDS.toMillis(30);
+  private static void waitForAdb(AndroidDebugBridge adb, long timeOutMs) {
+    // long timeOutMs = TimeUnit.SECONDS.toMillis(30);
```
Comment (edenman) [inline spoon-runner/src/main/java/com/squareup/spoon/SpoonUtils.java]: remove this line instead of commenting it out

**Comment by edenman** [comment]
@intelradoux sorry for the delay, i was on vacation.  one suggestion, otherwise lgtm.  also, can you please squash into a single commit?

**Comment by intelradoux** [comment]
Comments removed and commit sqashed.

**Comment by edenman** [comment]
Thanks, @intelradoux!

**Pull request merged**